start_cerberus.py

```diff
@@ -133,6 +134,24 @@ def main(cfg):
                              "assuming that the nodes are ready")
                 watch_nodes_status = True
 
+            # Monitor cluster operators status
+            if watch_operators:
+                operators_status = runcommand.invoke("oc get co -o yaml")
+                status_yaml = yaml.load(operators_status, Loader=yaml.FullLoader)
+                watch_operators_status, failed_operators = \
+                    kubecli.monitor_cluster_operators(iteration, status_yaml)
+                logging.info("Iteration %s: Cluster Operator status: %s"
```

Comment:
+1 to printing the aggregated status of all the operators when they are healthy and print the failed operators in case of failures like you mentioned in the commit message.